bytebuffer-collections/src/main/java/io/druid/collections/bitmap/MutableBitmap.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.collections.bitmap;
+
+import java.nio.ByteBuffer;
+
+/**
+ * This class is meant to represent a simple wrapper around a bitmap class.
+ */
+public interface MutableBitmap extends ImmutableBitmap
+{
+  /**
+   * Empties the content of this bitmap.
+   */
+  public void clear();
+
+  /**
+   * Compute the bitwise-or of this bitmap with another bitmap. The current
+   * bitmap is modified whereas the other bitmap is left intact.
+   * <p>
```

Comment:
These dangling `<p>`s apparently get in the way of the building of javadoc for Druid.  They should be cleaned up.  It should be acceptable to just replace `<p>` with a space.